### ISSUE TITLE
Equalizer support 

### ISSUE THREAD

**Issue by wasyster** [jfversluis/Plugin.Maui.Audio#59]
It would be nice to have a support for equalizer, or point me to a direction how to add one for Android and iOS.
I found that Android has an equalizer in Equalizer, but I can't figure it out how to wired up in the player.

**Comment by jfversluis** [comment]
I don't think we will add anything like this anytime soon ourselves. If anyone is up for the challenge, feel free to discuss the proposed API changes and open a PR.
